[2024-10-06T04:59:43.215+0000] {taskinstance.py:1087} INFO - Dependencies all met for <TaskInstance: pdf_extraction_with_dochorizon.upload_to_s3 manual__2024-10-06T04:54:24.192876+00:00 [queued]>
[2024-10-06T04:59:43.225+0000] {taskinstance.py:1087} INFO - Dependencies all met for <TaskInstance: pdf_extraction_with_dochorizon.upload_to_s3 manual__2024-10-06T04:54:24.192876+00:00 [queued]>
[2024-10-06T04:59:43.225+0000] {taskinstance.py:1283} INFO - 
--------------------------------------------------------------------------------
[2024-10-06T04:59:43.226+0000] {taskinstance.py:1284} INFO - Starting attempt 2 of 2
[2024-10-06T04:59:43.226+0000] {taskinstance.py:1285} INFO - 
--------------------------------------------------------------------------------
[2024-10-06T04:59:43.240+0000] {taskinstance.py:1304} INFO - Executing <Task(PythonOperator): upload_to_s3> on 2024-10-06 04:54:24.192876+00:00
[2024-10-06T04:59:43.243+0000] {standard_task_runner.py:55} INFO - Started process 4517 to run task
[2024-10-06T04:59:43.245+0000] {standard_task_runner.py:82} INFO - Running: ['***', 'tasks', 'run', 'pdf_extraction_with_dochorizon', 'upload_to_s3', 'manual__2024-10-06T04:54:24.192876+00:00', '--job-id', '217', '--raw', '--subdir', 'DAGS_FOLDER/PDF-Klippa.py', '--cfg-path', '/tmp/tmpw3748nm9']
[2024-10-06T04:59:43.247+0000] {standard_task_runner.py:83} INFO - Job 217: Subtask upload_to_s3
[2024-10-06T04:59:43.268+0000] {logging_mixin.py:137} WARNING - /home/***/.local/lib/python3.7/site-packages/***/settings.py:249 DeprecationWarning: The sql_alchemy_conn option in [core] has been moved to the sql_alchemy_conn option in [database] - the old setting has been used, but please update your config.
[2024-10-06T04:59:43.348+0000] {task_command.py:389} INFO - Running <TaskInstance: pdf_extraction_with_dochorizon.upload_to_s3 manual__2024-10-06T04:54:24.192876+00:00 [running]> on host c14bde59b801
[2024-10-06T04:59:43.464+0000] {taskinstance.py:1513} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=pdf_extraction_with_dochorizon
AIRFLOW_CTX_TASK_ID=upload_to_s3
AIRFLOW_CTX_EXECUTION_DATE=2024-10-06T04:54:24.192876+00:00
AIRFLOW_CTX_TRY_NUMBER=2
AIRFLOW_CTX_DAG_RUN_ID=manual__2024-10-06T04:54:24.192876+00:00
[2024-10-06T04:59:43.621+0000] {taskinstance.py:1772} ERROR - Task failed with exception
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/operators/python.py", line 175, in execute
    return_value = self.execute_callable()
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/operators/python.py", line 192, in execute_callable
    return self.python_callable(*self.op_args, **self.op_kwargs)
  File "/opt/airflow/dags/PDF-Klippa.py", line 145, in upload_to_s3
    dochorizon_s3_key = f'dochorizon/{os.path.basename(dochorizon_output_path)}'
  File "/usr/local/lib/python3.7/posixpath.py", line 146, in basename
    p = os.fspath(p)
TypeError: expected str, bytes or os.PathLike object, not NoneType
[2024-10-06T04:59:43.641+0000] {taskinstance.py:1327} INFO - Marking task as FAILED. dag_id=pdf_extraction_with_dochorizon, task_id=upload_to_s3, execution_date=20241006T045424, start_date=20241006T045943, end_date=20241006T045943
[2024-10-06T04:59:43.656+0000] {standard_task_runner.py:105} ERROR - Failed to execute job 217 for task upload_to_s3 (expected str, bytes or os.PathLike object, not NoneType; 4517)
[2024-10-06T04:59:43.699+0000] {local_task_job.py:159} INFO - Task exited with return code 1
[2024-10-06T04:59:43.727+0000] {taskinstance.py:2582} INFO - 0 downstream tasks scheduled from follow-on schedule check
